File: Day_01/solveB.py
latest = 0
states = [latest]
n = 0
sequence = [line.strip() for line in open('input.txt')]

while states.count(latest) != 2:
    latest = eval(str(latest) + sequence[n])
    states.append(latest)
    n += 1
    if n >= len(sequence):
        n = 0
print(latest)

File: 2018/Day_01/solveB.jl
latest = 0
states = [latest]
n = 1
sequence = [line for line in readlines(open("input.txt"))]

while count(x->x==latest, states) != 2
	global latest = eval(Meta.parse(string(latest,sequence[n])))
	append!(states, latest)
	global n += 1
	if n > length(sequence)
		global n = 1
	end
end

println(latest)
